plugins/datadog.js

```diff
@@ -0,0 +1,132 @@
+const _ = require('lodash');
+require('dotenv').config({silent: true});
+
+var core = require('../core.js');
+const ds = require('../datastore.js');
+
+function isCallable (text) {
+    return text.includes('datadog');
+}
+
+function executePlugin (channel, callback, text, user) {
```

Comment:
One style suggestion for this function:  this has multiple nested blocks and a healthy amount of lines in the function.  This is a good candidate for breaking up the functionality into smaller isolated functions